web/src/message_edit.ts

```diff
@@ -458,12 +503,12 @@ function edit_message($row, raw_content) {
             content: raw_content,
             file_upload_enabled,
             giphy_enabled: giphy_state.is_giphy_enabled(),
-            minutes_to_edit: Math.floor(realm.realm_message_content_edit_limit_seconds / 60),
+            minutes_to_edit: Math.floor(realm.realm_message_content_edit_limit_seconds ?? 0 / 60),
```

Comment:
`??` has low precedence.

```suggestion
            minutes_to_edit: Math.floor((realm.realm_message_content_edit_limit_seconds ?? 0) / 60),
```